lib/python/variable_init.cpp

```diff
@@ -329,5 +334,7 @@ Parameters
    Type of the variable's elements. Is deduced from other arguments
    in most cases. Defaults to ``sc.DType.float64`` if no deduction is
    possible.
+aligned:
```

Comment:
Unrelated to this change, but I just realize that sphinx-autodoc-typehints will not help us here (or any other function without Python wrapper)? It looks like the types are not documented: https://scipp.github.io/generated/classes/scipp.Variable.html#scipp.Variable.__init__